src/main/java/com/spikes2212/control/SmartMotionSettings.java

```diff
@@ -0,0 +1,58 @@
+package com.spikes2212.control;
+
+import java.util.function.Supplier;
+
+/**
+ * The smart motion constants used in a smart motion loop, in addition to the regular PID constants.
+ */
+public class SmartMotionSettings {
+
+    /**
+     * The minimum velocity of the motor during this smart motion loop.
```

Comment:
change the full sentence into a phrase

Reply:
This is a short summary, and therefore should consist of full sentences and not a short phrase.